src/openms/include/OpenMS/FORMAT/FileHandler.h

```diff
@@ -166,76 +205,222 @@ namespace OpenMS
 
       @param filename The name of the file to store the data in.
       @param exp The experiment to store.
+      @param allowed_types a @p vector<FileTypes::Files> containing the types supported where this function is called. If the vector is empty, load from any type that we have a handler for. Otherwise @getType() is called internally to check the type
```

Comment:
needs rewording, once the implementation logic is clear

Reply:
reworded